src/Form/Form.js

```diff
@@ -323,8 +362,14 @@ class Form {
         if (this.form.matches(selector)) {
             this.addInput(this.form)
         } else {
-            this.form.querySelectorAll(selector).forEach(input => this.addInput(input))
+            const foundInputs = this.form.querySelectorAll(selector)
```

Comment:
This is a further performance safeguard. If the form contains too many inputs, we stop adding more.